editor/editor_help.h

```diff
@@ -195,10 +197,19 @@ class EditorHelp : public VBoxContainer {
 	static String doc_version_hash;
 	static Thread worker_thread;
 
+	inline static bool script_docs_loaded = false;
```

Comment:
These are static inline. That means Editor help will get unique instances of these variables per cpp file that includes them. Is this intentional?

Reply:
Isn't `static inline` just a way to define static variable in a header, without having to declare it in header and define in cpp file? Or are they not equivalent?

Reply:
They're not equivalent. It's like inline functions. The cpp file (translation unit) reads it as "I get my own copy to optimize what I like"

Reply:
That's unfortunate, because they are used as equivalent in lots of places 💀

Reply:
Well. Seems static inline is a new c++17 feature. How ignorant I am!